fuzzers/manul/builder.Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+ARG parent_image=gcr.io/fuzzbench/base-builder
+FROM $parent_image
+
+RUN apt-get update && apt-get install -y python3 python3-pip
+
+RUN pip3 install psutil
+
+RUN git clone https://github.com/mxmssh/manul
+
+RUN cd / && wget http://lcamtuf.coredump.cx/afl/releases/afl-latest.tgz && \
+        tar xf afl-latest.tgz && mv afl-2.52b afl && cd /afl && make
+
+RUN apt-get update && \
```

Comment:
these two lines can be removed, it is already in base-builder.

apt-get update && \
apt-get install wget -y